pkgs/by-name/sv/svp/package.nix

```diff
@@ -40,48 +34,39 @@ let
   '';
 
   libraries = [
-    fakeLsof
-    ffmpeg.bin
-    glibc
-    zenity
-    libmediainfo
-    libsForQt5.qtbase
-    libsForQt5.qtwayland
-    libsForQt5.qtdeclarative
-    libsForQt5.qtscript
-    libsForQt5.qtsvg
-    libusb1
```

Comment:
I suggest keeping ffmpeg, libmediainfo and libusb1 in place. IIRC these are used by the transcoding and lighting features in SVP.